project/util/success_url.py

```diff
@@ -1,4 +1,4 @@
-from django.utils.http import is_safe_url
+from django.utils.http import url_has_allowed_host_and_scheme
```

Comment:
Looks like `is_safe_url` has been deprecated in favor of `url_has_allowed_host_and_scheme`, which I guess is more descriptive.  The old name probably sounded like more of a cure-all than it actually was.